src/libraries/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.cs

```diff
@@ -11,6 +11,8 @@
 using System.Security.AccessControl;
 using System.Text;
 
+#pragma warning disable CA2249 // Consider using 'string.Contains' instead of 'string.IndexOf'
```

Comment:
If we'd rather put this line close to the where the rule is actually triggered, let me know!

Reply:
Yes, we should surround the relevant code with:
```C#
#pragma warning disable CA2249
...
#pragma warning restore CA2249
```
That said, why does it need to be suppressed?